src/LICENSE.js

```diff
@@ -0,0 +1,3 @@
+/** @license
+__LICENSE__
```

Comment:
`rollup-plugin-replace` has been configured to replace `__LICENSE__` with the contents of `LICENSE` in the root of the project.

We may be able to look into defining a shortened version of the license, given that we don't need to display the entirety of the Apache 2.0 license. We could just say that it's licensed under Apache 2.0 with a link to view the license in the browser.